chapters/infrastructure/processing/reading/README.md

```diff
@@ -1,12 +1,163 @@
+By default, the input directory is the root of the project and the output directory is a directory with the same name under the build directory.
+In the snippet below, there are 3 steps that will run the `command`, `reveal_embed` and `docusaurus` plugins, in that order, with their respective default input and output directories.
+
+```yaml
+stages:
+  - make_assets
+  - embed_reveal
+  - docusaurus
+
+build_dir: /build
+output_dir: /output
+
+make_assets:
+  plugin: command
+# [...]
+
+embed_reveal:
+  plugin: reveal_embed
+# [...]
+
+docusaurus:
+  plugin: docusaurus
+# [...]
+```
+
+Using content that was built/generated in a previous step can be done by using the full path to that content.
+In the example below, we are using content generated in the `make_assets` step as an argument to the `static_assets` option of the `docusaurus` plugin.
+
+```yaml
+make_assets:
+  plugin: command
+  options:
+    command: make
+    locations:
+      - content/chapters/data/lecture
+      - content/chapters/io/lecture
+      # [...]
+    args:
+      - all
+
+# [...]
+
+docusaurus:
+  plugin: docusaurus
+  options:
+# [...]
+    static_assets:
+      - slides/Data: /build/make_assets/content/chapters/data/lecture/_site
+      - slides/IO: /build/make_assets/content/chapters/io/lecture/_site
+# [...]
+
+```
+
+#### Docusaurus
+
+Docusaurus requires a directory structure similar to this:
+```
+docs
+└── Labs
+    └── Compute
+        ├── media
+        ├── quiz
+        ├── overview.md
+        ├── processes.md
+        └── threads.md
+```
+This is different from the structure proposed in the [Developing and Organizing Content](../../../develop-organize/overview/reading/README.md) chapter.
+We have developed the `Docusaurus` plugin for the `Open Education Builder` in order to facilitate this reorganization.
+
+The most important option of this plugin is the `structure` option, which defines the order of the chapters and their sections and subsections, together with the source of the content.
+A simplified example, taken from the [Operating Systems](https://github.com/open-education-hub/operating-systems) repository, that maps to the directory structure above can be seen below.
+
+```yaml
+# [...]
+    structure:
+      - Lab:
+        - Compute:
+            path: content/chapters/compute/lab/content
+            extra:
+              - ../media
+              - ../quiz
+            subsections:
+              - Overview: overview.md
+              - Processes: processes.md
+              - Threads: threads.md
+# [...]
+```
+A section (such as `Lab` or `Compute` above) can either be followed by a list of subsections (as is the case for `Lab`) or by a few options.
+In the case of the `Compute` section, it has 3 options: `path`, `extra` and `subsections`.
+
+The `subsections` option specifies the subsections of the current section. These can be pages that will be rendered (as in this example) or other sections.
+
```

Comment:
```suggestion
In the case of the `Compute` section, it has 3 options:

- The `subsections` option specifies the subsections of the current section.
  These can be pages that will be rendered (as in this example) or other sections.
```
Use a list for these options rather than a paragraph as they are closely related.